### PR TITLE
[Redesign]Improve performance of expand/collapse large number of rows.

### PR DESCRIPTION
**Pull request by ryuyu**
Addresses https://github.com/NuGet/NuGetGallery/issues/4172

Switch to collapsing tbody instead of individual rows.

@joelverhagen @skofman1

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 1]
```diff
@@ -290,7 +290,11 @@
                             if (!packageVersion.Deleted && (packageVersion.Listed || (Model.IsOwner(User) || User.IsAdministrator())))
                             {
                                 rowCount++;
-                                <tr class="@(rowCount >= 6 ? "hidden-version collapse" : "")">
+                                if(rowCount >=6)
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: nit space after `>=`

Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: nit space after `if`

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 2]
```diff
@@ -326,7 +330,11 @@
                             else if (packageVersion.Deleted)
                             {
                                 rowCount++;
-                                <tr class="deleted @(rowCount >= 6 ? "hidden-version collapse" : "")">
+                                if(rowCount >=6)
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: same

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml (hunk 1 above)
Comment (scottbommarito) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Put `6` in a constant somewhere so we don't have to remember to change both lines of code if we decide we want a different number of rows uncollapsed.

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 3]
```diff
@@ -344,8 +352,8 @@
                 </table>
                 @if (rowCount > 5)
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: Can we put this in a constant somewhere as well, especially if it's reused?

File: src/NuGetGallery/Views/Statistics/_PivotTable.cshtml [hunk 4]
```diff
@@ -80,8 +80,25 @@
                                 <th scope="col">Downloads</th>
                             </tr>
                         </thead>
-                        <tbody data-bind="foreach: Table">
-                            <tr data-bind="attr: { class: $index() > 6 ? 'hidden-row collapse' : '' }">
+                        <tbody data-bind="foreach: ShownRows">
+                            <tr>
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Statistics/_PivotTable.cshtml]: Can we make this inner portion into its own template and reuse it here and below?

**Pull request merged**